h2gis-functions/src/test/java/org/h2gis/functions/io/geojson/GeojsonImportExportTest.java

```diff
@@ -568,5 +568,19 @@ public void testReadComplexFile() throws Exception {
         stat.execute("DROP TABLE IF EXISTS TABLE_POINTS_READ");
         stat.close();
     }
+
+    @Test
+    public void testReadProperties() throws Exception {
+        Statement stat = connection.createStatement();
+        stat.execute("DROP TABLE IF EXISTS TABLE_PROPERTIES_READ;");
+        stat.execute("CALL GeoJsonRead("+ StringUtils.quoteStringSQL(GeojsonImportExportTest.class.getResource("data.geojson").getPath()) + ", 'TABLE_PROPERTIES_READ');");
+        ResultSet res = stat.executeQuery("SELECT * FROM TABLE_PROPERTIES_READ;");
+        res.next();
+        assertEquals(-105576, res.getDouble(2), 0);
```

Comment:
You should test all the object from the ResultSet to ensure that the geojson have been well parsed.
In this test the property field `mean_pleasantness` is not parsed and the fields `first_measure_ISO_8601` and `last_measure_ISO_8601` should be parsed as timestamp and not string.